src/components/alerts/SubmitTranscriptAlert.tsx

```diff
@@ -146,30 +126,19 @@ const PromptStepOne = forwardRef<HTMLButtonElement, PromptOneProps>(
 PromptStepOne.displayName = "PromptStepOne";
 
 const PromptStepTwo = forwardRef<HTMLButtonElement, PromptTwoProps>(
-  ({ onCancel, prRepo, onSubmit }, ref) => {
+  ({ onCancel, prRepo, onSubmit }) => {
     return (
       <>
-        <AlertDialogHeader fontSize="lg" fontWeight="bold">
-          Let&apos;s review your edits
-        </AlertDialogHeader>
         <AlertDialogBody>
-          Are you sure you want to submit your review? This would create a PR on{" "}
+          {/* Are you sure you want to submit your review? This would create a PR on{" "} */}
```

Comment:
If we're settling for the new text, we can remove this.

Reply:
okay, wanted to confirm if this is what we want first